tools/rosgraph/src/rosgraph/roslogging.py

```diff
@@ -174,33 +174,28 @@ def __init__(self, colorize=True):
     def emit(self, record):
         level, color = _logging_to_rospy_names[record.levelname]
         record_message = _defaultFormatter.format(record)
-        if 'ROSCONSOLE_FORMAT' in os.environ.keys():
-            msg = os.environ['ROSCONSOLE_FORMAT']
-            msg = msg.replace('${severity}', level)
-            msg = msg.replace('${message}', str(record_message))
-            msg = msg.replace('${walltime}', '%f' % time.time())
-            msg = msg.replace('${thread}', str(record.thread))
-            msg = msg.replace('${logger}', str(record.name))
-            msg = msg.replace('${file}', str(record.pathname))
-            msg = msg.replace('${line}', str(record.lineno))
-            msg = msg.replace('${function}', str(record.funcName))
-            try:
-                from rospy import get_name
-                node_name = get_name()
-            except ImportError:
-                node_name = '<unknown_node_name>'
-            msg = msg.replace('${node}', node_name)
-            if self._get_time is not None and not self._is_wallclock():
-                t = self._get_time()
-            else:
-                t = time.time()
-            msg = msg.replace('${time}', '%f' % t)
-            msg += '\n'
+        msg = os.environ.get('ROSCONSOLE_FORMAT',
+                             '[${severity}] [${time}]: ${message}')
+        msg = msg.replace('${severity}', level)
+        msg = msg.replace('${message}', str(record_message))
+        msg = msg.replace('${walltime}', '%f' % time.time())
+        msg = msg.replace('${thread}', str(record.thread))
+        msg = msg.replace('${logger}', str(record.name))
+        msg = msg.replace('${file}', str(record.pathname))
+        msg = msg.replace('${line}', str(record.lineno))
+        msg = msg.replace('${function}', str(record.funcName))
+        try:
+            from rospy import get_name
+            node_name = get_name()
+        except ImportError:
+            node_name = '<unknown_node_name>'
+        msg = msg.replace('${node}', node_name)
+        if self._get_time is not None and not self._is_wallclock():
+            t = self._get_time()
         else:
-            msg = '[%s] [WallTime: %f]' % (level, time.time())
-            if self._get_time is not None and not self._is_wallclock():
-                msg += ' [%f]' % self._get_time()
-            msg += ' %s\n' % record_message
+            t = time.time()
+        msg = msg.replace('${time}', '%f' % t)
```

Comment:
In the case that simtime is being used, roscpp prints both the wall clock and the sim time, like this:

```
[INFO] [${walltime}, ${simtime}]: ${message}
```

Previously Python did:

```
[INFO] [WallTime: ${walltime}] [${simtime}] ${message}
```

With the ` [${simtime}]` part being excluded if simtime is not being used.

This patch doesn't work for me, since it no longer prints wall and sim time at the same time, it now prints wall or sim time. This is both different from the current behavior and different from roscpp (which is the goal of this pr as far as I gather).